libs/cloud/odc/aws/queue.py

```diff
@@ -180,3 +184,16 @@ def get_messages(
         raise Exception(f"Limit {limit} is not valid.")
 
     return itertools.islice(messages, limit)
+
+
+def publish_to_topic(action, stac):
+    """
+    Publish a message to an sns topic
+    """
+    sns = boto3.resource("sns")
+    sns_topic = sns.create_topic(Name=SNS_TOPIC_NAME)
```

Comment:
We'll have a topic already existing - `dea-public-data-sentinel-2`, and will want the tool to accept the topic name (or ARN [the Absolute Resource Name in AWS]) as a parameter - e.g., `--publish arn:aws:sns:ap-southeast-2:xxxxxxxxxxxxxxxxx:dea-public-data-sentinel-2` (where the xxx's are the account number, which I've forgotten)